### PR TITLE
shudarshon.chaks/enabled artifact and artifact checksum upload steps in circleci

### PR DESCRIPTION
**Pull request by shudarshon-binary**

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -87,6 +122,9 @@ jobs:
       - git_checkout_from_cache
       - npm_install
       - build
+      - compress
+      - upload_artifact
```
Comment (4p00rv) [inline .circleci/config.yml]:
```suggestion
       # Uploading the built code to s3 to create a backup of key services separate from Kubernetes deployment.
      - upload_artifact
```
Adding a comment to let the future developers know of the reason for this.

File: deploy/cron.sh [hunk 2]
```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env bash
```
Comment (4p00rv) [inline deploy/cron.sh]: @shudarshon-binary I don't think we need this file in the repo.

File: .circleci/config.yml [hunk 3]
```diff
@@ -52,18 +53,52 @@ commands:
...
+          name: "Compile project"
+          command: npm run build
+
+  compress:
+    description: "Compress"
+    steps:
+      - run:
+          name: "Compress"
+          command: |
+            pushd dist/compressed/
```
Comment (aminmarashi-binary) [inline .circleci/config.yml]: nitpicking: `pushd` without a `popd`. Better to use `cd`.

Comment (aminmarashi-binary) [reply .circleci/config.yml]: I see, used `OLDPWD` instead. :ok_hand:

File: .circleci/config.yml [hunk 4]
```diff
@@ -52,18 +53,52 @@ commands:
...
+            tar -cvf artifact.tar *
```
Comment (aminmarashi-binary) [inline .circleci/config.yml]: Why don't we compress the tar file? it needs `z` for `zip`.

Comment (aminmarashi-binary) [reply .circleci/config.yml]: We could also exclude `.git` directory here because the artifact won't use it anyway.

**Pull request merged**